project/Build.scala

```diff
@@ -1316,14 +1336,30 @@ object Build {
     ).
     settings(
       Compile / resourceGenerators += Def.task {
-        val jsDestinationFile = (Compile / resourceManaged).value / "dotty_res" / "scripts" / "scaladoc-scalajs.js"
-        sbt.IO.copyFile((`scaladoc-js` / Compile / fullOptJS).value.data, jsDestinationFile)
-        Seq(jsDestinationFile)
+        val contributorsFile = (`scaladoc-js-contributors` / Compile / fullOptJS).value.data
+        val contributorsDestinationFile = Paths.get("docs-for-dotty-page", "js", "contributors.js").toFile
```

Comment:
Why don’t we put this one into the `resourceManaged` path?

Reply:
It's not resource managed. If it was, it would be shipped with scaladoc, and we want it to be under `docs` directory, so it will be replaceable in context of static site